WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -158,16 +191,132 @@ public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bun
             actionBar.setTitle(R.string.invite_people);
         }
 
+        mInviteLinkContainer = rootView.findViewById(R.id.invite_links_container);
+        mShimmerContainer = rootView.findViewById(R.id.shimmer_view_container);
+        mGenerateLinksButton = rootView.findViewById(R.id.generate_links);
+        mLoadAndRetryLinksContainer = rootView.findViewById(R.id.load_and_retry_container);
+        mRetryButton = rootView.findViewById(R.id.get_status_retry);
+        mLoadingLinksProgress = rootView.findViewById(R.id.get_links_status_progress);
+        mManageLinksContainer = rootView.findViewById(R.id.manage_links_container);
+        mShareLinksButton = rootView.findViewById(R.id.share_links);
+        mLinksRoleTextView = rootView.findViewById(R.id.links_role);
+        mLinksRoleContainer = rootView.findViewById(R.id.links_role_container);
+        mDisableLinksButton = rootView.findViewById(R.id.disable_button);
+        mExpireDateTextView = rootView.findViewById(R.id.expire_date);
+
         return rootView;
     }
 
+    @Override public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        mViewModel = new ViewModelProvider(this, mViewModelFactory).get(PeopleInviteViewModel.class);
+
+        mGenerateLinksButton.setOnClickListener(v -> {
+            mViewModel.onGenerateLinksButtonClicked();
+        });
+
+        mShareLinksButton.setOnClickListener(v -> {
+            mViewModel.onShareButtonClicked(
+                    mLinksRoleTextView.getText() != null
+                            ? mLinksRoleTextView.getText().toString()
+                            : ""
+            );
+        });
+
+        mDisableLinksButton.setOnClickListener(v -> {
+            if (!isAdded()) return;
+
+            PeopleInviteDialogFragment
+                    .newInstance(this, DialogMode.DISABLE_INVITE_LINKS_CONFIRMATION)
+                    .show(getParentFragmentManager(), DIALOG_TAG);
+        });
+
+        mRetryButton.setOnClickListener(v -> {
+            mViewModel.onRetryButtonClicked();
+        });
+
+        mViewModel.getSnackbarEvents().observe(getViewLifecycleOwner(), event ->
+                event.applyIfNotHandled(holder -> {
+                    WPSnackbar.make(mCoordinator,
+                            mUiHelpers.getTextOfUiString(mContextProvider.getContext(), holder.getMessage()),
+                            Snackbar.LENGTH_LONG)
+                              .show();
+                    return Unit.INSTANCE;
+                })
+        );
+
+        mViewModel.getInviteLinksUiState().observe(getViewLifecycleOwner(), uiState -> {
+            manageLinksControlsVisibility(uiState);
+
+            manageShimmerSection(uiState.isShimmerSectionVisible(), uiState.getStartShimmer());
+
+            manageActionButtonsEnable(uiState.getEnableActionButtons());
```

Comment:
Just to be on a safe side of clarity, what do you think about modifying the name of `manageActionButtonsEnable` to `manageActionButtonsEnabledState` and `getEnableActionButtons` to `isEveryActionButtonEnabled`?